docs/install-new.md

```diff
@@ -0,0 +1,94 @@
+# Content
+- [Creating Application Gateway and Azure Kubernetes Cluster using Azure Resource Manager (ARM) template](#deploying-the-infrastructure)
+- [Setting up the Azure Kubernetes Cluster](#setting-up-the-azure-kubernetes-cluster)
+
+## Deploying the infrastructure
+
+To create the pre-requisite Azure resources, You can use the following template. It creates:
+1) Azure Virtual Network with 2 subnets.
+2) Azure Application Gateway v2
+3) Azure Kubernetes Service cluster with required permission to deploy nodes in the Virtual Network.
+4) User Assigned Identity to initialize the aad-pod-identity service and ingress controller.
+5) Set required RBACs.
+
+
+Steps:
+
+1) Create a service principal that will be assigned to aks cluster in the template.
+    ```bash
+    az ad sp create-for-rbac --skip-assignment
+    Note: appId and password.
+    az ad sp show --id <appId>
+    Note: objectId.
+    ```
+
+2) After the above, click to create a custom template deployment. Provide the appId for servicePrincipalClientId, password, objectId in the parameters.
+
+    <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fakshaysngupta%2Fapplication-gateway-kubernetes-ingress%2Fmaster%2Fdeploy%2Fazuredeploy.json" target="_blank">
+        <img src="http://azuredeploy.net/deploybutton.png"/>
+    </a>
+    <a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fakshaysngupta%2Fapplication-gateway-kubernetes-ingress%2Fmaster%2Fdeploy%2Fazuredeploy.json" target="_blank">
+        <img src="http://armviz.io/visualizebutton.png"/>
+    </a>
+
+    After the deployment completes, you can look at the Output window for parameters needed in the following steps.
+
```

Comment:
I think we should give a prescriptive example here, specifically hightlighting the "parameters" in sample log output.  
This will help the reader (removes the guesswork and possible config errors IMO)

Reply:
thanks for adding the output.